### DISCUSSION TITLE
ssh key authentication

### DISCUSSION THREAD

**Discussion by czosnkowy666** [mucommander/mucommander#602] [Q&A]
Is there a plan to support key authentication in sftp?

**Comment by czosnkowy666** [comment]
I see in SFTP connection option to provide key file but I was not able to make it work :( 
I'm on mac os version if it change anything 

I found that it is issue with jsch that is not maintained and not supporting new type of keys and new format of those 
There is fork that should handle it `com.github.mwiede:jsch:0.1.61`

**Reply by ahadas** [reply]
Thanks for checking this @czosnkowy666 
Yeah, we've replaced the previous library that was used with for sftp with jsch as the previous one was completely broken but yeah, there hasn't been a new version of jsch for quite a while, if there's a good fork (in terms of functionality and license) then it makes sense to switch  I'll convert this to an issue then
